docs/cookbook/configuring-a-message-bus.md

```diff
@@ -36,9 +36,14 @@ framework:
                 middleware:
                     - msgphp.messenger.console_message_receiver
             event_bus:
+                default_middleware: allow_no_handlers
                 middleware:
                     - msgphp.messenger.console_message_receiver
-                    - allow_no_handler
+
+services:
```

Comment:
this should be reverted, as the default behavior is described below. As well as this example snippet.

Reply:
yep, sorry